Gemfile

```diff
@@ -3,7 +3,11 @@
 source 'https://rubygems.org'
 
 group :backend, :frontend, :core, :api do
-  gemspec require: false
+  gem 'solidus_api', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_backend', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_core', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_frontend', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_sample', github: 'solidusio/solidus', glob: '**/*.gemspec'
```

Comment:
The glob format is a workaround for this issue: "Bundler cannot add multiple gems from the same git source" (https://github.com/rubygems/bundler/issues/7346).